engines/director/castmember/bitmap.cpp

```diff
@@ -682,6 +675,15 @@ void BitmapCastMember::setPicture(Image::ImageDecoder &image, bool adjustSize) {
 	setModified(true);
 }
 
+Common::Point BitmapCastMember::getRegistrationOffset() {
+	return Common::Point(_regX - _initialRect.left, _regY - _initialRect.top);
+}
+
+Common::Point BitmapCastMember::getRegistrationOffset(int16 width, int16 height) {
```

Comment:
I think the better names would be `currentWidth` and `currentHeight`, e.g. as I understand, these are the current dimensions, when the sprite was stretched.

Reply:
Fixed